sample_app/app/views/user_mailer/password_reset.html.erb

```diff
@@ -1,5 +1,10 @@
-<h1>User#password_reset</h1>
+<h1><%= t "users.reset_password.mailer.header" %></h1>
 
-<p>
-  <%= @greeting %>, find me in app/views/user_mailer/password_reset.html.erb
-</p>
+<p><%= t "users.reset_password.mailer.plaese_click_link" %></p>
+
+<%= link_to t("users.reset_password.mailer.reset_link"),
+  edit_password_reset_url(@user.reset_token, email: @user.email) %>
```

Comment:
anh dùng edit_password_reset_path thay edit_password_reset_url được không

Reply:
Dạ được chứ anh, nhưng thay vì nhật được việc send mail thì mình nhận dc cái lỗi to bự chảng trên màn hình ý.
Vì *_url sẽ trả về full url
Còn *_path chỉ trả về đường dẫn tương đối.

ANh giải thích thêm cho em chỗ này đi :D